include/ProgramVariable.h

```diff
@@ -0,0 +1,45 @@
+#ifndef PROGRAM_VARIABLE_H
+#define PROGRAM_VARIABLE_H
+
+#include "RunAnalysis.h"
+#include "Utils.h"
+
+class ProgramVariable {
+private:
+  Value *value;
+
+  // name is llvm IR name without % or @
+  std::string cleanedName;
+
+  // name is llvm IR name maybe with % or @
+  std::string rawName;
+
+  std::list<ProgramVariable> aliases;
```

Comment:
Use a comment here to specify which kind of aliases (may or must) these are, and whether the list is an over or underapproximation.

I think this is an underapproximation of must aliases (that's what the RLC for Java does). That is to say, every entry in this list is definitely an alias of the variable being represented, but there might be other aliases that are not in the list.

Reply:
It aliases the same way AliasMap did, and you're correct in that they're underapproximations of must aliases. I'll add that comment to the code